### ISSUE TITLE
[Dev] Use Local Okta User data instead of contacting Okta when populating affiliations

### ISSUE THREAD

**Issue by knollfear** [CMSgov/eAPD#3090]
Feature -  
When the affiliations are fetched from the database we are currently requesting some additional data from Okta.  Once #3089 is complete we can and should pull that data from the local DB table.  For affiliations where we have okta user data we should include that data in query result when finding affiliations.  For users that we do not have Okta data for we should ping Okta and store that data for the next time.  

### This task is done when…
This could include:
 - [ ] requesting an affiliation does not contact Okta unless we do not have the data locally
 - [ ] Data for users that we do not have is added to the DB when fetched from Okta.

**Comment by thetif** [comment]
duplicate of #3091
